docs/configuration.md

```diff
@@ -54,15 +54,17 @@
+| Environment Variable                            | Required  | Default                                      | Description                                                                                                                                                                                                                                            |
+|-------------------------------------------------|-----------|----------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `OPENCLARITY_AWS_REGION`                        | **yes**   |                                              | Region where the Scanner instance needs to be created                                                                                                                                                                                                  |
+| `OPENCLARITY_AWS_SUBNET_ID`                     | **yes**   |                                              | SubnetID where the Scanner instance needs to be created                                                                                                                                                                                                |
+| `OPENCLARITY_AWS_SECURITY_GROUP_ID`             | **yes**   |                                              | SecurityGroupId which needs to be attached to the Scanner instance                                                                                                                                                                                     |
+| `OPENCLARITY_AWS_KEYPAIR_NAME`                  |           |                                              | Name of the SSH KeyPair to use for Scanner instance launch                                                                                                                                                                                             |
+| `OPENCLARITY_AWS_SCANNER_IMAGE_NAME_FILTER`     |           | `ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-*` | The Name filter used for finding AMI to instantiate Scanner instance. See: [DescribeImages](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html)                                                                            |
+| `OPENCLARITY_AWS_SCANNER_IMAGE_OWNERS`          |           | `099720109477`                               | Comma separated list of OwnerID(s)/OwnerAliases used as Owners filter for finding AMI to instantiate Scanner instance. See: [DescribeImages](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html)                           |
```

Comment:
what if we keep the AMI ID and only default to image name/owner filter if its not specified? would this be feasable?

Reply:
same for GCP and Azure